packages/web3-core/src/types.ts

```diff
@@ -37,3 +38,18 @@ export type Web3BaseProviderConstructor = new <API extends Web3APISpec>(
 	url: string,
 	net?: Socket,
 ) => Web3BaseProvider<API>;
+
+export interface ConfigOptions {
```

Comment:
Earlier it was a private type for the `Web3Config` class then the context was fine. As we are exporting now, would be nice to rename it to `Web3ConfigOptions`